force-app/main/default/lwc/givingSummaryDetailPage/givingSummaryDetailPage.js-meta.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>52.0</apiVersion>
+    <isExposed>true</isExposed>
+    <targets>
+        <target>lightning__RecordPage</target>
+    </targets>
+    <targetConfigs>
+      <targetConfig targets="lightning__RecordPage">
+          <objects>
+              <object>Contact</object>
```

Comment:
I was under the impression this component was only going to be used at Experience level.

Reply:
We changed the idea as there were some issues in unit testing. So we decided to do this component environment agnostic and wrap it for displaying on the detail page. Later we will wrap it to another component to display it in an Experience cloud